lib/cosmos/gui/choosers/telemetry_chooser.rb

```diff
@@ -44,6 +42,14 @@ class TelemetryChooser < Qt::Widget
     # Label for the item
     attr_accessor :item_label
 
+    # Custom error which we raise and catch
+    class NotFoundError < StandardError
```

Comment:
I created a custom error so we aren't just catching everything and masking other unexpected problems